docs/template_plugin/tests/functional/op_reference/acosh.cpp

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <vector>
+
+#include "base_reference_test.hpp"
+
+using namespace ngraph;
+
+namespace {
+
+template <typename T>
+std::vector<T> create_iota_vector(size_t size, T first_value = {}) {
+    std::vector<T> d(size);
+    std::iota(begin(d), end(d), first_value);
+    return d;
+}
```

Comment:
We should consider adding such helper functions to kind of test utils/helpers file to be reusable between tests.

Reply:
Yes we should. If someone else will use it it might move it to `base_reference_test.hpp`. I don't want to put it there because now test are in global namespace and some name might have collision.